### PR TITLE
VACMS-7322: Structure centralized content to support Cerner medical records action links

### LINKED ISSUE #7322: Structure centralized content to support Cerner medical records action links
## User Story or Problem Statement

As a Veteran at a Cerner-supported VAMC, i want the medical records link to take me to the right system.

As a national editor of centralized content, i want to be able to edit the Cerner link and have it update all VAMCs that use Cerner, from one place.

As a local editor, i want to understand the centralized content and how it may show up on the FE, depending on my VAMC's Cerner status. 

<details><summary>Mural showing various experiences</summary>

[See mural here](https://app.mural.co/t/vagov6717/m/vagov6717/1639530168395/b114bb336a1db8c0a571272b10a230e1360d2775?sender=kevinwalsh8610)

Cursor_and_Medical_records_CTA_for_Cerner_VAMCs_•_VFS-CMS

</details>

## Acceptance Criteria
On Centralized content node
- [x] Update centralized content: one field for Cerner, one field for non-Cerner

On VAMC Medical Records page content type
- [x] Create EFF field to pull in the Cerner CTA right below the react widget
- [x] Update field labels (react widget OR cerner CTA, depending on the system)

Also
- [x] Address incorrect placement of the "Medical records office locations", which should come _before_ "Get your records by mail or fax".

<details><summary>Screenshot</summary>

Create_VAMC_System_Medical_Records_Office___VA_gov_CMS

</details>

## Implementation steps

Field and form config on Medical Records Office content type

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [x] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [x] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [x] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
closes #7322
Implementing @kevwalsh 's config from this tugboat: https://main-eyjs0sncpqkjhpxgnkuhzayuve040hg6.demo.cms.va.gov/admin/config/development/configuration

## Testing done
Visual / behat

## Screenshots
image

## QA steps
 - [x] as admin go to admin/content/audit/centralized-content-paragraphs and validate the view exists. 
 - [x] As Ryan.stubblebine@va.gov, go to `/node/add/vamc_system_medical_records_offi`
 - [x] Visually verify presence of updated label `Get your records online` -> `React widget (for non-Cerner VAMCs)`
 - [x] Visually verify presence of new Action link below `health-records` (per screenshot)
 - [x] Visually verify `Medical records office locations` appears above `Get your records by mail or fax`

### CODE REVIEW

File: config/sync/core.entity_form_display.node.vamc_system_medical_records_offi.default.yml [hunk 1]
```diff
@@ -33,7 +34,6 @@ third_party_settings:
     group_edit:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.vamc_system_medical_records_offi.default.yml:36]: AWOL

File: tests/behat/drupal-spec-tool/content_model_paragraph_fields.feature [hunk 2]
```diff
@@ -104,7 +104,6 @@ Feature: Content model: Paragraph fields
 | Paragraph type | Service location | Use the facility's hours | field_hours | List (text) | Required | 1 | Select list |  |
 | Paragraph type | Service location | Other phone numbers | field_phone | Entity reference revisions |  | 5 | Inline entity form - Complex - Table View Mode |  |
 | Paragraph type | Service location | Use the general facility phone number | field_use_main_facility_phone | Boolean |  | 1 | Single on/off checkbox |  |
-| Paragraph type | Service location | Hours | field_office_hours | Office hours |  | Unlimited | Office hours (week) |  |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_paragraph_fields.feature:107]: I don't think this should be here.

File: tests/behat/drupal-spec-tool/content_model_vocabulary_fields.feature [hunk 3]
```diff
@@ -13,7 +13,7 @@ Feature: Content model: Vocabulary fields
 | Vocabulary | Sections | Description | field_description | Text (plain) |  | 1 | Textfield |  |
 | Vocabulary | Sections | Product | field_product | Entity reference |  | 1 | Select list |  |
 | Vocabulary | VHA health service taxonomy | Common conditions | field_commonly_treated_condition | Text (plain) |  | 1 | Textfield |  |
-| Vocabulary | VHA health service taxonomy | Health Service API ID | field_health_service_api_id | Text (plain) | Required | 1 | Textfield |  |
+| Vocabulary | VHA health service taxonomy | Health Service API ID | field_health_service_api_id | Text (plain) |  | 1 | Textfield |  |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_vocabulary_fields.feature:16]: I don't think this should be changed.

File: tests/behat/drupal-spec-tool/content_model_vocabulary_fields.feature [hunk 4]
```diff
@@ -23,6 +23,5 @@ Feature: Content model: Vocabulary fields
 | Vocabulary | VHA health service taxonomy | Service description | field_vet_center_service_descrip | Text (plain, long) |  | 1 | Text area (multiple rows) |  |
 | Vocabulary | VHA health service taxonomy | Type of Care | field_vet_center_type_of_care | List (text) |  | 1 | Select list |  |
 | Vocabulary | VHA health service taxonomy | This is a required Vet Center service | field_vet_center_required_servic | Boolean |  | 1 | Single on/off checkbox |  |
-| Vocabulary | VHA health service taxonomy | Enforce unique combo service | field_enforce_unique_combo_servi | Allow Only One |  | 1 | Allow Only One widget |  |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_vocabulary_fields.feature:26]: I don't think this should be going away.

File: tests/behat/drupal-spec-tool/views.feature [hunk 5]
```diff
@@ -271,5 +272,3 @@ Feature: Views
 | PDF Audit | Default | default | Default |
 | PDF Audit | PDF Audit | pdf_audit | Page |
 | PDF Audit | Data export | pdf_audit_export | Data export |
-| Facility Services | Accordion audit | accordion_audit | Page |
-| Facility Services | Accordion audit export | accordion_audit_export | Data export |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/views.feature:275]: Should not be exiting the building.

File: tests/behat/drupal-spec-tool/views.feature [hunk 6]
```diff
@@ -258,8 +261,6 @@ Feature: Views
 | Rich Text Field Audit | Default | default | Default |
 | Rich Text Field Audit | Content Audit - Rich Text Fields | content_audit_rich_text | Page |
 | Rich Text Field Audit | Data export | rich_text_audit_data_export | Data export |
-| Rich Text Field Audit | Content Audit - Buttons | content_audit_buttons | Page |
-| Rich Text Field Audit | Data export | buttons_export | Data export |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/views.feature:262]: Should not be going away.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Some unexpected changes especially related to behat tests.   
I think you got bit by someone else cutting off the tip of the spear in the spec tool.

File: config/sync/core.entity_form_display.node.vamc_system_medical_records_offi.default.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.node.vamc_system_medical_records_offi.default.yml:36]: shame---->Ethan<----shame

File: tests/behat/drupal-spec-tool/views.feature [hunk 7]
```diff
@@ -11,93 +11,99 @@ Feature: Views
 | Administration section | administration_section | Taxonomy terms | Enabled | Top-level items in the Section taxonomy |
```
Comment (ethanteague) [inline tests/behat/drupal-spec-tool/views.feature:11]: @swirtSJW bunch of stuff got shuffled when I exported out of the spec tool. Just a heads up that these aren't manual changes.

Comment (swirtSJW) [reply tests/behat/drupal-spec-tool/views.feature:11]: Thanks for the heads up.  Fortunately the test will let me know if it's all good.  ;)

**Review by swirtSJW — APPROVED** [review]
Changes look good.   New dev support View looks good.   Obviates the need for Devel to find the pid.
image
Node fields  good
image

**Comment by swirtSJW** [comment]
The three failing AXE tests were due to timeouts and the three pages that timed out are not the one affected by this PR
image

**Pull request merged**